galaxy.yml

```
@@ -1,14 +1,48 @@
-namespace: release-engineering
+### REQUIRED
+# The namespace of the collection. This can be a company/brand/organization or product namespace under which all
+# content lives. May only contain alphanumeric lowercase characters and underscores. Namespaces cannot start with
+# underscores or numbers and cannot contain consecutive underscores
+namespace: release_engineering
+
+# The name of the collection. Has the same character restrictions as 'namespace'
 name: dockpulp_ansible
-version: "{{ version }}"
-description: Ansible modules to create and update dockpulp repo
+
+# The version of the collection. Must be compatible with semantic versioning
+version: master
```

I'm not sure `master` is compatible with semver. What about 0.1.0?
```suggestion
version: 0.1.0
```

---

Yes, `master` is incompatible. Thank you!

Below are some comments from Andrew Hills in jira, I add them here as a later reference:

According to this document:

https://galaxy.ansible.com/docs/contributing/version.html#versioning-collections

which links to this document:

https://docs.ansible.com/ansible/devel/dev_guide/developing_collections_distributing.html#collection-versions

it needs to have X.Y.Z format as per this standard:

https://semver.org/

Down in the FAQ section, they have this suggestion:

> How should I deal with revisions in the 0.y.z initial development phase?

> The simplest thing to do is start your initial development release at 0.1.0 and then increment the minor version for each subsequent release.

https://semver.org/#how-should-i-deal-with-revisions-in-the-0yz-initial-development-phase